pkg/storage/trb_guidance_letter_recommendation.go

```diff
@@ -287,10 +290,10 @@ func (s *Store) UpdateTRBGuidanceLetterRecommendationOrder(
 		WITH new_positions AS (
 			SELECT *
 			FROM json_to_recordset(:newPositions)
-			AS new_positions (id uuid, position_in_letter int)
+			AS new_positions (id uuid, position_in_letter int, category guidance_recommendation_category_type)
 		)
 		UPDATE trb_guidance_letter_recommendations
-		SET position_in_letter = new_positions.position_in_letter
+		SET position_in_letter = new_positions.position_in_letter, category = new_positions.category
```

Comment:
I'm not sure if we should be _updating_ the category as part of this operation or not. I think the intent here might be to allow only updating order within a certain category, but right now you could probably pass in a bunch of IDs in different categories and this query (as I understand it) would update their category to all be the same (whatever's passed in to the `update` param)

Reply:
i was hoping this would instead update the category to the (potentially new) incoming category for each of the update items. we do allow category updates, so i am thinking of that

Reply:
this has been updated! thanks Clay